src/styles/utilits/_mixin.scss

```diff
@@ -0,0 +1,9 @@
+@mixin centerDotSize ($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin positionOfHand ($hand-height) {
+  top: calc(50% - #{$hand-height});
+  left: calc(50% - (#{$hand-width} / 2));
+}
```

Comment:
But you don't need to use calc in scss

Reply:
Sometimes yes, but here i have different units of measurement (px and %), so without calc i cannot make a calculation.